src/tests/conformance/query/proof_path.rs

```diff
@@ -0,0 +1,74 @@
+use tempfile::TempDir;
+
+use crate::{
+    protocol::{
+        codecs::binary::{BinaryMessage, Payload},
+        proto::{TmLedgerMapType, TmProofPathRequest, TmProofPathResponse},
+    },
+    setup::node::{Node, NodeType},
+    tools::{rpc::wait_for_ledger_info, synth_node::SyntheticNode},
+};
+
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn c025_TM_PROOF_PATH_REQUEST_TM_PROOF_PATH_RESPONSE_node_should_respond_to_proof_path_req() {
```

Comment:
Maybe making it a little bit shorter.
Maybe we don't need to explicitly mention "`proof path`" in the description since we have the `c025_TM_PROOF_PATH_REQUEST_TM_PROOF_PATH_RESPONSE` test prefix.
```suggestion
async fn c025_TM_PROOF_PATH_REQUEST_TM_PROOF_PATH_RESPONSE_send_req_expect_rsp() {
```

Reply:
Done